star-ros2/README.md

```
@@ -64,6 +64,28 @@ map (RTAB-Map - global, corrected)
+source /workspaces/STAR/star-ros2/install/local_setup.bash  # after first build
+
+# Set RMW
+export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+
+# Build everything (proto gen + colcon)
+cd /workspaces/STAR
+./build-ros2.sh
+```
+
+**Required apt package** (for `safety_monitor.launch.py`):
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)